app/src/components/search/AreaSelect.vue

```diff
@@ -11,12 +11,12 @@
     @change="onInputChange"
   >
     <template v-slot:item="{ item }">
-      <img class="areaImage" :src="item.img" />
+      <img class="areaImage" alt="areaImage" :src="item.img" />
```

Comment:
"areaImage" is not descriptive. We should set the alt-tag to something like "Deutsche Flagge" and "Erdkugel" depending on the selectecd item.